concourse/scripts/pipelines-cloudfoundry.sh

```diff
@@ -29,6 +29,11 @@ debug: ${DEBUG:-}
 EOF
 }
 
+generate_manifest_file() {
+   # This exists because concourse does not support boolean value interpolation by design
+   local enable_auto_deploy=$([ "${ENABLE_AUTO_DEPLOY:-}" ] && echo "true" || echo "false")
```

Comment:
shellcheck didn't like this:
```
In ./concourse/scripts/pipelines-cloudfoundry.sh line 34:
   local enable_auto_deploy=$([ "${ENABLE_AUTO_DEPLOY:-}" ] && echo "true" || echo "false")
         ^-- SC2155: Declare and assign separately to avoid masking return values.
```